images.CI/measure-provisioners-duration.ps1

```diff
@@ -0,0 +1,121 @@
+param(
+    [string]$PackerLogPath = "$env:HOME/Downloads/temp2/packer-mac.txt",
```

Comment:
why is it `packer-mac.txt`? should it be `packer-log.txt` instead?

Reply:
It should be empty by default and Mandatory.
I have done this change locally but forgot to do commit before push :(
Fixed